### DISCUSSION TITLE
Welcome to Mario-Remake Discussions!

### DISCUSSION THREAD

**Discussion by seannkelleyy** [seannkelleyy/Mario-Remake#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by seannkelleyy** [comment]
2-6-24 Meeting Notes:
Room:    THO 051
Date:    Tuesday
Time:    1:30 PM to 4:00 PM

Old Business:
Interface PR needs to be merged into main. Preferably before the meeting but definitely during.

New Business:
Discuss new tickets (#18 and #19).

Help anyone who needs help with current tickets.

Send an email to the TA's to schedule a meeting.

Send an email to Kirby to schedule code review if we haven't already.

Feel free to comment or edit with other ideas.

**Reply by jliet359** [reply]
Review Code Smells and create a list of 10 of them.
